libbeat/processors/actions/drop_fields.go

```diff
@@ -62,25 +67,54 @@ func newDropFields(c *conf.C) (processors.Processor, error) {
 		}
 	}
 
-	f := &dropFields{Fields: config.Fields, IgnoreMissing: config.IgnoreMissing}
+	// Parse regexp containing fields and removes them from initial config
+	regexpFields := make([]match.Matcher, 0)
+	for i := len(config.Fields) - 1; i >= 0; i-- {
+		field := config.Fields[i]
+		if strings.HasPrefix(field, "/") && strings.HasSuffix(field, "/") && len(field) > 2 {
+			config.Fields = append(config.Fields[:i], config.Fields[i+1:]...)
+
+			matcher, err := match.Compile(field[1 : len(field)-1])
+			if err != nil {
+				return nil, fmt.Errorf("wrong configuration in drop_fields. %w", err)
```

Comment:
I'd rather extend this error message to make it easier to debug. The test would need an update too.

```suggestion
				return nil, fmt.Errorf("wrong configuration in drop_fields[%d]=%s. %w", i, field, err)
```

Reply:
oki sure